README.md

```diff
@@ -68,19 +68,28 @@ You will also need a Promise polyfill for [older browsers](http://caniuse.com/#f
 We recommend [taylorhakes/promise-polyfill](https://github.com/taylorhakes/promise-polyfill)
 for its small size and Promises/A+ compatibility.
 
-For use with webpack, add this package in the `entry` configuration option
+For use with webpack, either add this package in the `entry` configuration option
 before your application entry point:
 
 ```javascript
 entry: ['whatwg-fetch', ...]
 ```
 
-For Babel and ES2015+, make sure to import the file:
+... or require the file:
+
+```javascript
+require('whatwg-fetch')
```

Comment:
Is `require()` something specific to webpack? I'm not a webpack user.

Reply:
It's not specific to webpack but it's part of the common usage. I pointed it out as it is another way to load the package, in addition to the first one that was included in the docs.